### ISSUE TITLE
More flexible (customizable) metadata

### ISSUE THREAD

**Issue by tkurze** [opencast-ilias/OpenCast#181]
**Is your feature request related to a problem? Please describe.**
It is related to an organizational (but not to a technical) problem: the „old“ Studer+Raimann Plugin always sets the rights holder meta-data field to the value: users_lastname, users_firstname (user_email). The community plugin is way more flexible compared to the old plugin; but the old behavior can’t be mimicked. For us this is a problem, because there is software in place, that (partly) relies upon the fact, that the rights holder field contains the data is described before.
Also we have another issue, that – if solved – also remedies the first problem described above: we like to be able to use the course ID as a pre fill value; that would allow us, to link the video directly to a course in the campus management system.

**Describe the solution you'd like**
In my opinion, pre-fill data for metadata fields should be a) more flexible than just a dropdown select box and b) there should be more options to choose from.

Regarding a) Instead of a dropdown, a textbox in which placeholders may be used seems way more flexible; as multiple values can be combined and ‚formatted‘ together; for example (excl. ><): >${user.last_name}, ${user.first_name} (${user.mail})<.

Regarding b)
Options to choose from resp. variables to use may for example include: course number (reference number), course name, user email, user name, user title, ...

**Describe alternatives you've considered**
Just add the course ID as a possible pre fill value for metadata fields. Not as flexible as the primary proposal, but would still kind of fix our problem.

There was a problem with our Ilias Test system when a took the screenshots; please ignore the *MISSING* strings, etc.
**Additional context**
![7TJq0HGrSDPZbpJB](https://github.com/opencast-ilias/OpenCast/assets/1611618/cc56067e-fe87-487f-a1f0-5413a24f4b77)

**Comment by reiferschris** [comment]
@tkurze we are working on solving this issue with @ferishili 
There will be a PR soon and we would be very happy if you could also test the enhancement. We will tag you once the PR is finished
